cloud/google/clusteractuator.go

```diff
@@ -15,21 +15,30 @@ package google
 
 import (
 	"fmt"
+	"github.com/golang/glog"
 	"golang.org/x/net/context"
 	"golang.org/x/oauth2/google"
 	compute "google.golang.org/api/compute/v1"
 	"sigs.k8s.io/cluster-api/cloud/google/clients"
+	gceconfigv1 "sigs.k8s.io/cluster-api/cloud/google/gceproviderconfig/v1alpha1"
 	clusterv1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
 	client "sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/typed/cluster/v1alpha1"
 )
 
+const (
+	firewallRuleAnnotationPrefix = "gce.clusterapi.k8s.io/firewall"
+	firewallRuleInternalName = "-allow-cluster-internal"
```

Comment:
consider renaming it to firewallRuleInternalSuffix to make it clear that it is meant to be concatenated to something else

Reply:
done.